test/decorateWithLogger.test.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const tap = require('tap')
+const test = tap.test
+const fastify = require('fastify')({ logger: true })
+const plugin = require('../')
+
+fastify.register(plugin, { addHook: false, keys: new Set(['123456']) })
+
+test('verifyBearerAuth', (t) => {
+  t.plan(1)
+  fastify.ready(() => {
+    t.ok(fastify.verifyBearerAuth)
+  })
+})
+
+test('verifyBearerAuthFactory', (t) => {
+  t.plan(1)
+  fastify.ready(() => {
+    t.ok(fastify.verifyBearerAuthFactory)
+  })
+})
```

Comment:
```suggestion
const { test } = require('tap')
const fastify = require('fastify')
const plugin = require('..')
```
decorators already have their own tests inside `decorate.test.js`

Reply:
> Good work @songpr !
> 
> Just a few nits but LGTM with the suggestions applied.

The main thing that is missing in decorate is there is no test case when log is enable.
I duplicate the test just to make sure that the test will still be passed when log is enabled.

Reply:
I covered those cases within the suggestions :)